ironfish/src/logger/reporters/console.ts

```diff
@@ -28,14 +28,34 @@ export const loggers: Record<logType, typeof console.log> = {
   silent: silentLogger,
 }
 
-export class ConsoleReporter extends TextReporter {
-  getConsoleLogger(logType: logType): typeof console.log {
-    const logger = loggers[logType]
-    Assert.isNotUndefined(logger)
-    return logger
+export const getConsoleLogger = (logType: logType): typeof console.log => {
+  const logger = loggers[logType]
+  Assert.isNotUndefined(logger)
+  return logger
+}
+
+export const logObjToJSON = (logObj: ConsolaReporterLogObject): string => {
+  const objectArgs: Record<string, unknown>[] = logObj.args.filter(
+    (a) => typeof a === 'object',
+  ) as Record<string, unknown>[]
+  const otherArgs = logObj.args.filter((a) => typeof a != 'object')
+
+  const toLog = {
+    ...objectArgs[0],
+    level: logObj.level,
+    tag: logObj.tag,
+    date: logObj.date,
+    message: otherArgs.join(' '),
   }
 
+  return JSON.stringify(toLog)
```

Comment:
You could potentially use `IJSON.stringify` here to avoid any potential issues if you pass in a bigint as an argument to the logger. There may be a try/catch preventing a crash though, I haven't looked to see

Reply:
Or to clarify, I think it'd be nice to use IJSON.stringify here so we don't have to refactor all of these logs -- there's a good chance someone could add a breaking log in the future since I expect most developers won't be running this flag enabled in production